docs/INSTALL.md

```diff
@@ -158,34 +158,29 @@ This will create custom fields named (`instanceId`, `ssh_key` & `userData`) as s
 
 ### Configurable Options
 
-The module will automatically create Configurable Options when you create your first product. You can adjust these options as needed.
+The module will automatically create Configurable Options when you create your first product. You can select option groups that apply to your product.
 
-#### If No Custom Plan is Selected
+To adjust these options:
 
-1. Go to System Settings -> Configurable Options and click on edit
+1. Go to System Settings -> Configurable Options
 2. Edit the newly created Options
 
-![Configurable Options](images/configurable_options.png)
+![Configurable Options](images/configurable_options_normal.png)
 
+Through these configurations you can:
 
-3. For Configurable Option "Location - template|Template", adjust the prices for Windows OS
+- Set custom pricing for backup options
+- Configure available locations
+- Configure available custom plans
+- Adjust prices for available images (such as Windows)
+- Backup and Location: set custom pricing for backup options, and configure available locations
 
-![OS prices](images/os_prices.png)
+If custom plans are utilised:
```

Comment:
Could note here that custom plans are an account feature that requires special provisioning, contact support etc.

Reply:
Added!